tests/vault_action_tests_base.py

```diff
@@ -0,0 +1,77 @@
+from st2tests.base import BaseActionTestCase
+
+from tests.utils import get_config_file_path
+from tests.utils.hvac_integration_test_case import HvacIntegrationTestCase
+
+
+class VaultActionTestCase(HvacIntegrationTestCase, BaseActionTestCase):
+    dummy_pack_config = None
+    secret_v1 = False
+    secret_v2 = False
+    default_token_lease = "1h"
+
+    # in setUp() and tearDown(), explicitly handle super()-like calls.
+    #  - HvacIntegrationTestCase does not call super().
+    #  - BaseActionTestCase does call super().
+
+    def setUp(self):
+        HvacIntegrationTestCase.setUp(self)
```

Comment:
Let's use `super()` and MRO here instead of calling this explicitly.

```suggestion
        # Calls HvacIntegrationTestCase.setUp(self)
        super().setUp()
```